CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ For experimental package changes, see the [experimental CHANGELOG](experimental/
 ### :bug: (Bug Fix)
 
 * fix(instrumentation-http): fixed description for http.server.duration metric [#3710](https://github.com/open-telemetry/opentelemetry-js/pull/3710)
+* fix(opentelemetry-sdk-trace-web): don't crash in runtimes where location isn't defined [#3715](https://github.com/open-telemetry/opentelemetry-js/pull/3715)
```

Comment:
This isn't a huge deal, so if it's better to scrap this and wait for the larger refactor to not have Fetch autoinstrumentation depend on this package that's totally fine by me

Reply:
Simple enough change I'm fine to accept it